frontend/src/metabase/query_builder/components/filters/modals/InlineValuePicker/utils.ts

```diff
@@ -0,0 +1,24 @@
+import type Field from "metabase-lib/lib/metadata/Field";
+import type Filter from "metabase-lib/lib/queries/structured/Filter";
+
+import { isString } from "metabase/lib/schema_metadata";
+
+const fieldWidth = {
+  small: "11rem",
+  medium: "20rem",
+  full: "100%",
+};
+
+export const getFieldWidth = (field: Field, filter: Filter) => {
```

Comment:
This encapsulates the logic for detemrining field width

1. because `is` and `is not` are multi-token fields, they should be full width to acommodate as many tokens as possible
2. single-input operators (like contains) should be longer for strings than numbers
3. single-input number fields (like less than) should be the smallest

![Screen Shot 2022-08-01 at 6 33 00 PM](https://user-images.githubusercontent.com/30528226/182267203-6c8ab49f-a75f-4886-bcdd-54c270ad2f23.png)